notebooks/keras-with-openvino-backend/keras-with-openvino-backend.ipynb

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
<p>I think it is good to mention how these data can be obtained (using tokenizer?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>